evolution-chain/runtime/Cargo.toml

```diff
@@ -98,8 +97,6 @@ std = [
 	"sp-transaction-pool/std",
 	"sp-version/std",
 	"substrate-wasm-builder",
-	"fp-account/std",
-	"pallet-bridge-grandpa/std",
```

Comment:
why are these removed?

Reply:
they are not a dependency anymore

Reply:
`pallet-bridge-grandpa` is there at line 57

Reply:
pallet-bridge-grandpa was erroneous removed